### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by sigma-andex**
**Description of the change**

Clearly and concisely describe the purpose of the pull request. If this PR relates to an existing issue or change proposal, please link to it. Include any other background context that would help reviewers understand the motivation for this PR.

---

**Checklist:**

- [ ] Added a file to CHANGELOG.d for this PR (see CHANGELOG.d/README.md)
- [ ] Added myself to CONTRIBUTORS.md (if this is my first contribution)
- [ ] Linked any existing issues or proposals that this pull request should close
- [ ] Updated or added relevant documentation
- [ ] Added a test for the contribution (if applicable)

### CODE REVIEW

File: tests/purs/bundle/ObjectShorthand.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-var foo = 1;
+export var foo = 1;
```
Comment (thomashoneyman) [inline tests/purs/bundle/ObjectShorthand.js:1]: Why was this change necessary?

Comment (sigma-andex) [reply tests/purs/bundle/ObjectShorthand.js:1]: i think it gets removed due to optimization

Comment (thomashoneyman) [reply tests/purs/bundle/ObjectShorthand.js:1]: Huh, that doesn't sound good. But this is in the bundler, so 👌

**Review by thomashoneyman — APPROVED** [review]

File: tests/purs/bundle/ObjectShorthand.js (hunk 1 above)
Comment (sigma-andex) [reply tests/purs/bundle/ObjectShorthand.js:1]: yes that seems to me like a bug in the bundler, but I’m not sure
